src/hyp3_opera_rtc/prep_rtc.py

```diff
@@ -167,6 +167,7 @@ def prep_rtc(
         'output_dir': str(output_dir),
         'dual_pol': dual_pol,
         'resolution': int(resolution),
+        'data_validity_start_date': '20140403',
```

Comment:
By the way, where does `20140403` come from? I would have expected it to match the `2016-04-14` date that we chose for the [HyP3 validator](https://github.com/ASFHyP3/hyp3/blob/develop/apps/api/src/hyp3_api/validation.py#L251).